package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "pretest": "cross-env NODE_ENV=test && cross-env NODE_ENV=test npm run migrate:undo && cross-env NODE_ENV=test npm run migrate",
+    "pretest": "npm run migrate:undo && npm run migrate && npm run seed",
```

Comment:
Why did you remove cross-env here?

Reply:
I must have removed it while resolving merge conflicts. Will fix that right away

Reply:
Now fixed.